lib/yggdrasil/command/move.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Yggdrasil.Command.MoveCommand do
+  defstruct exit: nil
+end
+
+defimpl Yggdrasil.Command, for: Yggdrasil.Command.MoveCommand do
+  alias Yggdrasil.Message
+
+  def execute(cmd, player) do
+    Yggdrasil.Player.notify player, Message.info("You head towards #{cmd.exit}")
```

Comment:
Do we really want the execute to be aware of the player? Not sure it needs to be... feels out of place, I'll make a note down below where it's called and explain why I think so.